lib/pure/endians.nim

```diff
@@ -98,16 +126,919 @@ else:
+    assert fromBytesLE( uint64, buf, 3) == 0xdeadbeefcafebabe'u64
+    assert fromBytesLE(float32, buf, 3) == 0xcafebabe'f32
+    assert fromBytesLE(float64, buf, 3) == 0xdeadbeefcafebabe'f64
+
+    toBytesBE(i8 ,  i8_out.addr); assert  i8_out == i8
+    toBytesBE(u8 ,  u8_out.addr); assert  u8_out == u8
+    toBytesBE(i16, i16_out.addr); assert i16_out == i16_rev
+    toBytesBE(u16, u16_out.addr); assert u16_out == u16_rev
+    toBytesBE(i32, i32_out.addr); assert i32_out == i32_rev
+    toBytesBE(u32, u32_out.addr); assert u32_out == u32_rev
+    toBytesBE(i64, i64_out.addr); assert i64_out == i64_rev
+    toBytesBE(u64, u64_out.addr); assert u64_out == u64_rev
+    toBytesBE(f32, f32_out.addr); assert f32_out == f32_rev
+    toBytesBE(f64, f64_out.addr); assert f64_out == f64_rev
+
+    toBytesBE(i8 , buf, 3); assert fromBytesBE(   int8, buf, 3) == i8
+    toBytesBE(u8 , buf, 3); assert fromBytesBE(  uint8, buf, 3) == u8
+    toBytesBE(i16, buf, 3); assert fromBytesBE(  int16, buf, 3) == i16
+    toBytesBE(u16, buf, 3); assert fromBytesBE( uint16, buf, 3) == u16
+    toBytesBE(i32, buf, 3); assert fromBytesBE(  int32, buf, 3) == i32
+    toBytesBE(u32, buf, 3); assert fromBytesBE( uint32, buf, 3) == u32
+    toBytesBE(i64, buf, 3); assert fromBytesBE(  int64, buf, 3) == i64
+    toBytesBE(u64, buf, 3); assert fromBytesBE( uint64, buf, 3) == u64
+    toBytesBE(f32, buf, 3); assert fromBytesBE(float32, buf, 3) == f32
+    toBytesBE(f64, buf, 3); assert fromBytesBE(float64, buf, 3) == f64
+
+    toBytesLE(i8 ,  i8_out.addr); assert  i8_out == i8
+    toBytesLE(u8 ,  u8_out.addr); assert  u8_out == u8
+    toBytesLE(i16, i16_out.addr); assert i16_out == i16
+    toBytesLE(u16, u16_out.addr); assert u16_out == u16
+    toBytesLE(i32, i32_out.addr); assert i32_out == i32
+    toBytesLE(u32, u32_out.addr); assert u32_out == u32
+    toBytesLE(i64, i64_out.addr); assert i64_out == i64
+    toBytesLE(u64, u64_out.addr); assert u64_out == u64
+    toBytesLE(f32, f32_out.addr); assert f32_out == f32
+    toBytesLE(f64, f64_out.addr); assert f64_out == f64
+
+    toBytesLE(i8 , buf, 3); assert fromBytesLE(   int8, buf, 3) == i8
+    toBytesLE(u8 , buf, 3); assert fromBytesLE(  uint8, buf, 3) == u8
+    toBytesLE(i16, buf, 3); assert fromBytesLE(  int16, buf, 3) == i16
+    toBytesLE(u16, buf, 3); assert fromBytesLE( uint16, buf, 3) == u16
+    toBytesLE(i32, buf, 3); assert fromBytesLE(  int32, buf, 3) == i32
+    toBytesLE(u32, buf, 3); assert fromBytesLE( uint32, buf, 3) == u32
+    toBytesLE(i64, buf, 3); assert fromBytesLE(  int64, buf, 3) == i64
+    toBytesLE(u64, buf, 3); assert fromBytesLE( uint64, buf, 3) == u64
+    toBytesLE(f32, buf, 3); assert fromBytesLE(float32, buf, 3) == f32
+    toBytesLE(f64, buf, 3); assert fromBytesLE(float64, buf, 3) == f64
+
+
+  # NimVM tests
+  # -----------
+  # Most float32 tests are commented out because they crash the VM.
+  # This is a known limitation, the VM only supports float64 currently.
```

Comment:
This should be fixed by now.

Reply:
Indeed, all tests are passing now with the head version.